### ISSUE TITLE
Is it possible to fit scan mesh without landmark?

### ISSUE THREAD

**Issue by dragonylee** [Rubikplayer/flame-fitting#48]
Thanks for your work!
I am using your code `fit_scan.py` to fit a mesh with arbitrary topology. It's hard to generate landmarks for all inputs. Is there any way to fit it without landmarks please?

**Comment by bbakpil** [comment]
Hi, did you get any ideas/solutions?

**Comment by dragonylee** [comment]
> Hi, did you get any ideas/solutions?

No, so when I was comparing them, I explained that their method requires providing landmarks.

**Comment by TimoBolkart** [comment]
Fitting the model to a scan minimizes follows some iterative closest point scheme, which alternates between computing correspondences between each scan vertex and the FLAME model mean, and minimizing the distance between these points in correspondence. As ICP-based methods converge only locally die to the unknown correspondence, a good initialization is required for this optimization. One common way is to assume the correspondence for few keypoints to be given. Without these keypoints, you will need to find some other way to initialize the optimization to get good initial correspondences for the ICP to work.
